test-lit/lit.cfg.py

```diff
@@ -42,8 +26,8 @@
 
 
 # suffixes: A list of file extensions to treat as test files.
-config.suffixes = ['.f', '.F', '.FOR', '.for', '.f77', '.f90', '.F90', '.f95',
-                   '.fpp', '.FPP']
+config.suffixes = ['.f', '.F', '.FOR', '.for', '.f77', '.f90', '.F90',
```

Comment:
Having a look in tools/f18/f18.cc it seems we support the below extensions. Suggest we match this list here and not the list from old flang.

`        if (suffix == "f" || suffix == "F" || suffix == "ff" ||
            suffix == "f90" || suffix == "F90" || suffix == "ff90" ||
            suffix == "f95" || suffix == "F95" || suffix == "ff95" ||
            suffix == "cuf" || suffix == "CUF" || suffix == "f18" ||
            suffix == "F18" || suffix == "ff18") {
`

Reply:
In this list is missing FOR or for. Should they also be in the list?

Reply:
This set of supported suffixes will change over time so we just need to pick a good initial value. In that sense, any of the suggestions so far are probably good enough, we just need to chose our favourite. So how to do that? This list is the common set of common suffixes that are the default for all test directories. test directories might also add their own additional suffixes for unusual test inputs only seen in that directory. 

I think there are two logical ways to decide the initial set:
1) We add only the suffixes that we are already using in current tests and we grow it as needed.
2) We add all the suffixes that F18 understands as valid input.

I have a mild preference for the latter in which case we go with the list in my comment and yes we won't support .FOR, .for, .f77, .fpp or .FPP. 

That said, I am happy to accept an alternative proposal with decent justification.